tests/test_numpy_array.py

```diff
@@ -125,6 +125,13 @@ def test_at(arr):
     assert all(m.mutate_at_t(arr, 1, 0).ravel() == [1, 2, 4, 5, 5, 6])
 
 
+def test_subscript_via_call_operator(arr):
```

Comment:
Could you please try (untested):

```
@pytest.mark.parametrize("fn", [m.at_t, m.const_subscript_via_call_operator_t])
def test_const_elem_reference(fn):
    assert fn(arr, 0, 2) == 3
    assert fn(arr, 1, 0) == 4


@pytest.mark.parametrize("fn", [m.mutate_at_t, m.subscript_via_call_operator_t])
def test_mutable_elem_reference(fn):
    assert all(fn(arr, 0, 2).ravel() == [1, 2, 4, 4, 5, 6])
    assert all(fn(arr, 1, 0).ravel() == [1, 2, 4, 5, 5, 6])
```

Reply:
(I just renamed `fn` to `func`)

Reply:
adapted and pushed (i removed the redundant ones since you just do this parametrize thing)